src/sbvr-api/translations.ts

```diff
@@ -24,46 +24,68 @@ export type AliasValidNodeType =
 	| UnknownTypeNodes
 	| NullNode;
 const aliasFields = (
-	abstractSqlModel: AbstractSqlModel,
+	fromAbstractSqlModel: AbstractSqlModel,
+	toAbstractSqlModel: AbstractSqlModel,
 	resourceName: string,
 	aliases: Dictionary<string | AliasValidNodeType>,
 ): SelectNode[1] => {
-	const fieldNames = abstractSqlModel.tables[resourceName].fields.map(
+	const fromFieldNames = fromAbstractSqlModel.tables[resourceName].fields.map(
 		({ fieldName }) => fieldName,
 	);
-	const nonexistentFields = _.difference(Object.keys(aliases), fieldNames);
+	const nonexistentFields = _.difference(Object.keys(aliases), fromFieldNames);
 	if (nonexistentFields.length > 0) {
 		throw new Error(
 			`Tried to alias non-existent fields: '${nonexistentFields.join(', ')}'`,
 		);
 	}
-	return fieldNames.map(
+	const toFieldNames = toAbstractSqlModel.tables[resourceName].fields.map(
```

Comment:
Needs to check in the translatedResource name
```suggestion
	const toFieldNames = toAbstractSqlModel.tables[toResource].fields.map(
```